psa-crypto/tests/mod.rs

```diff
@@ -12,23 +12,22 @@ mod key_agreement;
 
 #[test]
 fn generate_integration_test() {
+    let mut usage_flags: UsageFlags = Default::default();
+    usage_flags.set_sign_hash(false);
+    usage_flags.set_verify_hash(false);
+    usage_flags.set_verify_message(true);
```

Comment:
everything should be set to `false` by default so only `verify_message` is necessary here! But it's fine, no need to change